apps/fipsinstall.c

```diff
@@ -64,6 +64,7 @@ const OPTIONS fipsinstall_options[] = {
     {"noout", OPT_NO_LOG, '-', "Disable logging of self test events"},
     {"corrupt_desc", OPT_CORRUPT_DESC, 's', "Corrupt a self test by description"},
     {"corrupt_type", OPT_CORRUPT_TYPE, 's', "Corrupt a self test by type"},
+    { "config", OPT_CONFIG, '<', "The parent config to verify"},
```

Comment:
Nit: there seems to be a spurious extra space after the open brace on this line, compared to the other lines.